configure.ac

```diff
@@ -490,6 +494,11 @@ if test x"$enable_module_schnorrsig" = x"yes"; then
   enable_module_extrakeys=yes
 fi
 
+if test x"$enable_module_schnorrsig_halfagg" = x"yes"; then
+  SECP_CONFIG_DEFINES="$SECP_CONFIG_DEFINES -DENABLE_MODULE_SCHNORRSIG_HALFAGG=1"
+  enable_module_schnorrsig=yes
+fi
```

Comment:
You must do this before testing for schnorrsig because the setting `enable_module_schnorrsig=yes` happens _after_ checking for `enable_module_schnorrsig=yes`. There's a comment above mentioning this (sorry, hard to find):
```
# Besides testing whether modules are enabled, the following code also enables
# module dependencies. The order of the tests matters: the dependency must be
# tested first.
```